kategory/src/main/kotlin/kategory/instances/IdInstances.kt

```diff
@@ -41,3 +41,14 @@ interface IdInstances :
             fa.ev().value
 
 }
+
+object IdSemigroupK : SemigroupK<Id.F>, GlobalInstance<SemigroupK<Id.F>>() {
+    override fun <A> combineK(x: HK<Id.F, A>, y: HK<Id.F, A>): HK<Id.F, A> = y
+}
+
+class IdMonoidK(val a: Any) : MonoidK<Id.F>, GlobalInstance<MonoidK<Id.F>>() {
```

Comment:
These 2 IdInstances (`IdSemigroupK` and `IdMonoidK`) were created to able to provide 
`WriterT.semigroupK(Id, IdSemigroupK)` and `WriterT.monoidK(Id, IdMonoidK(0))` for law tests from `WriterTTests`, and also  `StateT.semigroupK(Id, IdSemigroupK),` from `StateTTests`. 

I needed an instance of `SemigroupK<Id.F>` and `MonoidK<Id.F>` there. I agree that it does not have a very clear sense and I would completely agree on switching to a different Monad for those law tests if you can provide an Idea of which one to use.